src/mixins/ui/lasso-event-constants.js

```diff
@@ -13,6 +13,31 @@ export const LassoShapeEventConstants = {
 }
 
 export const LassoGlobalEventConstants = {
+  /**
+   * triggered when a new lasso tool is activated by pressing its
+   * corresponding button in the lasso tool set ui
+   */
+  LASSO_TOOL_TYPE_ACTIVATED: "lasso:tool:activated",
```

Comment:
new signals. Still could use a way to document the properties of the object returned to callbacks.